util/kubeconfig/kubeconfig_test.go

```diff
@@ -73,3 +81,108 @@ func TestGetKubeConfigSecret(t *testing.T) {
 		t.Fatalf("Expected found secret to be equal to input")
 	}
 }
+
+func getTestCACert(key *rsa.PrivateKey) (*x509.Certificate, error) {
```

Comment:
copy pasta from https://github.com/kubernetes-sigs/cluster-api/blob/f7e048646ae89c6e0ddf755c3760f233017386d9/bootstrap/kubeadm/internal/cluster/certificates.go#L448
couldn't import the internal package even if we export that `func`